### PR TITLE
Fixed bug of automatically logging out on refreshing the page

### LINKED ISSUE #227: A bug in the login form
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
Currently, the user log's out just when the page is refreshed

**Expected behavior:**
Logged in even after refreshing

**Steps to reproduce:**
Log-in and then refresh the page

Your operating system
OSX

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #227 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
This fixes the bug of automatically logging out when the page was refreshed

#### Changes proposed in this pull request:
The bug is fixed by adding a check when the window is loaded

### CODE REVIEW

**Comment by stealthanthrax** [comment]
@zamhaq @ms10398 @gabru-md , please have a look

File: src/settings.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>SUSI Login Portal</title>
```
Comment (zamhaq) [inline src/settings.html]: fix the indentation problems

**Comment by stealthanthrax** [comment]
@zamhaq , fixed it

File: src/js/login.js [hunk 2]
```diff
@@ -9,6 +9,21 @@ var accessToken = "";
 var time = "";
 var BASE_URL = "https://api.susi.ai";
 
+window.onload = () => {
+        chrome.storage.sync.get("loggedUser", function (obj) {
+        if(obj.loggedUser)
+            {
+                showLoggedInBlock(true);
+                console.log("logged in");
```
Comment (gabru-md) [inline src/js/login.js]: Remove console.logs

**Comment by stealthanthrax** [comment]
done @gabru-md

**Pull request merged**